packages/server/lib/controllers/config.controller.ts

```diff
@@ -59,6 +59,22 @@ const separateFlows = (flows: NangoSyncConfig[]): FlowConfigs => {
     );
 };
 
+const findPotentialUpgrades = (enabledFlows: NangoSyncConfig[], publicFlows: NangoSyncConfig[]) => {
+    return enabledFlows.map((flow) => {
+        const publicFlow = publicFlows.find((publicFlow) => publicFlow.name === flow.name);
+        if (!publicFlow) {
+            return flow;
+        }
```

Comment:
Can `publicFlow` be undefined or null in a `find`?

Reply:
Very possible. If it is a custom sync there will be no corresponding `publicFlow`